cpp/src/arrow/compute/exec/aggregate_node.cc

```diff
@@ -393,7 +422,11 @@ class GroupByNode : public ExecNode {
     // Execute aggregate kernels
     for (size_t i = 0; i < agg_kernels_.size(); ++i) {
       util::tracing::Span span;
-      START_SPAN(span, "Aggregate", {{"function", aggs_[i].function}});
+      START_SPAN(span, aggs_[i].function,
+                 {{"function", aggs_[i].function},
```

Comment:
We don't record this attribute in function.cc, we should be consistent about this.